internal/policies/apparmor/apparmor.go

```diff
@@ -222,14 +245,18 @@ func filesFromEntry(e entry.Entry, apparmorPath string) ([]string, error) {
 
 		profileFilePath := filepath.Join(apparmorPath, profile)
 		info, err := os.Stat(profileFilePath)
-		if errors.Is(err, os.ErrNotExist) {
-			return nil, fmt.Errorf(i18n.G("apparmor profile %q doesn't exist in SYSVOL apparmor/ subdirectory"), profile)
+		if err != nil {
```

Comment:
hum, we can have other kinds of errors that triggers this behavior (can’t read/stat because of parent directory), maybe the returned error should say "no accessible" rather than does not exist?